### PR TITLE
Add file path support for dxCreate3DImage

### PR DESCRIPTION
**Pull request by ERAGON007**
Added file path support for dxCreate3DImage function

### CODE REVIEW

File: Core/3dimage.lua [hunk 1]
```diff
@@ -23,6 +23,16 @@ function dgsCreate3DImage(...)
...
+
+	if type(img) == "string" then
+		local tex = dxCreateTexture(img)
+		if (tex) then
+			img = tex
+		else
+			error(dgsGenAsrt(img, "dgsCreate3DImage", 4, "string", "", "Required file path or texture"))
+		end
+	end
+
```
Comment (ERAGON007) [inline Core/3dimage.lua:35]: Display an error with a correct and helpful message

**Review by ERAGON007 — COMMENTED** [review]
Display new error message

**Review by xLive — CHANGES_REQUESTED** [review]
This will cause a memory leak because the texture doesn't get destroyed,

I think the best idea to use this:
https://github.com/thisdp/dgs/blob/fb8d462da2f939a6d2444669f9ea1c649a3efa6e/Core/image.lua#L56

**Comment by xLive** [comment]
> 
> 
> Display new error message

No need for this if the image is nil dgsCreate3DImage will draw a rectangle instead like [dgsCreateImage](https://github.com/thisdp/dgs/blob/master/Core/image.lua)

**Pull request merged**